src/Core/src/Platform/Windows/ControlExtensions.cs

```diff
@@ -21,6 +21,12 @@ public static void UpdateFont(this Control nativeControl, Font font, IFontManage
 		public static void UpdateBackgroundColor(this Control nativeControl, Color color, UI.Xaml.Media.Brush? defaultBrush = null) =>
 			nativeControl.Background = color.IsDefault && defaultBrush != null ? defaultBrush : color.ToNative();
 
+		public static void UpdateBackgroundColor(this Border nativeControl, Color color, UI.Xaml.Media.Brush? defaultBrush = null) =>
```

Comment:
Should this whole file get moved into ViewExtensions.cs?

Reply:
moved